src/tests/unit/pizzas-serializer.test.js

```diff
@@ -15,6 +15,55 @@ const {
   goalSerializedPizza,
 } = serializePizzaData;
 
+const testSerializedPizza = (serializedPizza, ...includedArgs) => {
+  const serializedPizzaData = 'data' in serializedPizza ? serializedPizza.data : serializedPizza;
+  serializedPizzaData.attributes.should.deep.equal(goalSerializedPizza.data.attributes, 'it sets attributes correctly');
+
+  serializedPizzaData.id.should.be.a('string', 'it sets type of id correctly');
+  serializedPizzaData.id.should.equal('1', 'it sets value of id correctly');
+  serializedPizzaData.type.should.equal('pizza', 'it sets value of type correctly');
+
+  serializedPizza.links.self.should.equal('/v1/pizzas/1', 'it sets top level link correctly');
+
+  if (includedArgs.includes('dough')) {
+    if ('included' in serializedPizza) {
+      serializedPizza.included
+        .should.deep.include(goalSerializedPizza.included[0], 'it adds dough to included');
+    }
+
+    serializedPizzaData.relationships.dough
+      .should.deep.equal(goalSerializedPizza.data.relationships.dough, 'it sets dough relationship values correctly');
+  } else {
+    should.equal(serializedPizzaData.relationships.dough.data, null, 'it puts null in serializedPizza.doughs');
+
+    if ('included' in serializedPizza && includedArgs.includes('ingredients')) {
+      serializedPizza.included.length.should.equal(1, 'it has only one element in `included`');
+
+      serializedPizza.included[0]
+        .should.deep.equal(goalSerializedPizza.included[1], 'it has an ingredient in `included`');
+    }
+  }
+
+  if (includedArgs.includes('ingredients')) {
+    if ('included' in serializedPizza) {
+      serializedPizza.included
+        .should.deep.include(goalSerializedPizza.included[1], 'it adds ingredients to included');
+    }
+
+    serializedPizzaData.relationships.ingredients
+      .should.deep.equal(goalSerializedPizza.data.relationships.ingredients, 'it sets ingredients relationship values correctly');
+  } else {
+    serializedPizzaData.relationships.ingredients.data.length.should.equal(0, 'it puts an empty array in serializedPizza.ingredients');
+
+    if ('included' in serializedPizza && includedArgs.includes('dough')) {
+      serializedPizza.included.length.should.equal(1, 'it has only one element in `included`');
+
+      serializedPizza.included[0]
+        .should.deep.equal(goalSerializedPizza.included[0], 'it has a dough in `included`');
+    }
+  }
+};
+
```

Comment:
moved common logic for checking if a resource was serialized correctly into this function